webapp/pages/profile/_id/_slug.vue

```diff
@@ -144,16 +144,17 @@
                     delay: { show: 500 },
                   }"
                   class="ds-tab-nav-item pointer ds-tab-nav-item-active"
-                  @click="tabActivity('posts', $event)"
                 >
-                  <ds-space margin="small">
-                    <!-- TODO: find better solution for rendering errors -->
-                    <no-ssr>
```

Comment:
@ogerly could you check if `no-ssr` is really necessary here?

Reply:
> no-ssr

@roschaefer  I was just reading. 
no-ssr is practically not necessary.